WillTrem/UInnovate
# 168 i18n add languages

Ability to add a new language dynamically that is reflected on the database and the front-end.

To test the PR please do the following

1. Open the add language modal, add a language, and click on save
<img width="813" alt="Screenshot 2024-01-13 at 12 09 35 AM" src="https://github.com/WillTrem/UInnovate/assets/74876532/a83be9d5-9e67-42a8-81fb-7c42e1b86760">

2. You should notice how a new column will be generated next to the labels column following the format LANG:<language_code>
<img width="578" alt="Screenshot 2024-01-13 at 12 11 12 AM" src="https://github.com/WillTrem/UInnovate/assets/74876532/112568cb-3d7a-42e7-ae66-25919d289faf">

3. Check that the changes are reflected in the db on the i18n_languages table and the i18n_translations view
<img width="525" alt="Screenshot 2024-01-13 at 12 12 32 AM" src="https://github.com/WillTrem/UInnovate/assets/74876532/171cccfa-ad6c-40eb-8180-5d7115f5df5b">
<img width="627" alt="Screenshot 2024-01-13 at 12 12 48 AM" src="https://github.com/WillTrem/UInnovate/assets/74876532/009b556e-36e5-4440-93d6-9bed997dee85">
